sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_sentiment_with_opinion_mining_async.py

```diff
@@ -94,67 +61,75 @@ async def analyze_sentiment_with_opinion_mining(self):
             credential=AzureKeyCredential(key)
         )
 
-        print("In this sample we will be combing through the reviews of a potential hotel to stay at: Hotel Foo.")
+        print("In this sample we will be a hotel owner going through reviews of their hotel to find complaints.")
 
         print(
-            "I first found a handful of reviews for Hotel Foo. Let's see if I want to stay here."
+            "I first found a handful of reviews for my hotel. Let's see what we have to improve."
         )
 
         documents = [
-            "The food and service were unacceptable, but the concierge were nice",
-            "The rooms were beautiful but dirty. The AC was good and quiet, but the elevator was broken",
-            "The breakfast was good, but the toilet was smelly",
-            "Loved this hotel - good breakfast - nice shuttle service.",
-            "I had a great unobstructed view of the Microsoft campus"
+            """
+            The food and service were unacceptable, but the concierge were nice.
+            After talking to them about the quality of the food and the process to get room service they refunded
+            the money we spent at the restaurant and gave us a voucher for near by restaurants.
+            """,
```

Comment:
I like the format in this sample as it is clear where each document starts and ends. Could you apply it in other places i.e. recognize_entities_azync?

Reply:
Having it like this introduces whitespace that looks weird printed, so I only have it like this in samples where the documents themselves won't be printed